index.html

```diff
@@ -69,5 +69,130 @@ <h2 class="connect">LET'S CONNECT</h2>
 				</section>
 				<span class="header-shape"></span>
 			</main>
+      <section class="work">
+        <div class="cardWork sec">
+          <picture id="snapshot"></picture>
```

Comment:
Kindly, the PICTURE tag is used to wrap various versions of images, low quality, other formats compatible with other browsers, etc. In this case, you must use the "IMG" tag and indicate the attribute (example) alt="main logo"